api/tests/unit/audit/test_unit_audit_signals.py

```diff
@@ -99,15 +108,16 @@ def test_send_audit_log_event_to_grafana__project_grafana_config__calls_expected
     project: Project,
 ) -> None:
     # Given
-    grafana_config = GrafanaProjectConfiguration(base_url="test.com", api_key="test")
-    project.grafana_config = grafana_config
     audit_log_record = AuditLog.objects.create(
         project=project,
         related_object_type=RelatedObjectType.FEATURE.name,
     )
     grafana_wrapper_mock = mocker.patch("audit.signals.GrafanaWrapper", autospec=True)
     grafana_wrapper_instance_mock = grafana_wrapper_mock.return_value
 
+    grafana_config = GrafanaProjectConfiguration(base_url="test.com", api_key="test")
+    project.grafana_config = grafana_config
```

Comment:
Having added the logic to prevent the postpone decorator from spawning unmanaged threads we need to move this here to prevent the generation of the audit log record itself from triggering an external request to Grafana (or Dynatrace as the case may be below).